docs/feature-flagging/mutual_exclusion.md

```diff
@@ -0,0 +1,61 @@
+# Mutual exclusion (Layers)
+
+:::note
+Layers are currently in closed Beta
+:::
+
+There are situations when you want to run concurrent experiments on the same surface. Eppo offers Layers as a option to keep your experiments mutual exclusive.
+
+Research from Microsoft has shown that in practice [interaction effects are vanishingly rare](https://www.microsoft.com/en-us/research/group/experimentation-platform-exp/articles/a-b-interactions-a-call-to-relax/). Given this, we recommend only using mutual exclusion when the overlap of two new treatments critically degrades the user experience. For instance, a design team might want to run a homepage redesign test while a marketing team might want to experiment on the headline messaging on the same page. If the new layout removes the messaging altogether or makes it unreadable, these experiments should clearly not overlap.
+
+## Creating a Layer
+
+Navigate to the Configuration section and click the "Create" button and select "Layer". You'll be promoted to fill out a number of details including the Layer name, the layer key, parameters, and the default variation.
+
+![Create a layer](/img/feature-flagging/create-layer.png)
+
+### Parameters
+Parameters are attributes that are changed in different Variations you plan to test within the Layer. They are also configured in code and can accpet the values you provide.
```

Comment:
```suggestion
Parameters are attributes that are changed in different variations you plan to test within the Layer. They are also configured in code and can accpet the values you provide.
```

I think we should stick to lowercase for variation, layer, feature flag, etc. Otherwise The Text Will Start Looking Pretty Weird Quickly